META

```diff
@@ -0,0 +1,36 @@
+description = "Coq Proof Assistant Plugin API"
+version = "dev"
+
+requires = "camlp5"
```

Comment:
This seems fishy, since camlp4 is also supported. I guess META should be (partly) generated by configure.